local-interchaintest/src/tests.rs

```diff
@@ -0,0 +1,80 @@
+use super::util;
+use serde_json::Value;
+use std::error::Error;
+
+const ERROR_MARGIN_PROFIT: u64 = 100000;
+
+pub fn test_profitable_arb(arbfile: Value) -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    let arbs = arbfile.as_array().expect("no arbs in arbfile");
+
+    util::assert_err("!arbs.is_empty()", arbs.is_empty(), false)?;
+
+    let profit: u64 = arbs
+        .iter()
+        .filter_map(|arb_str| arb_str.as_str())
+        .filter_map(|arb_str| {
+            serde_json::from_str::<Value>(arb_str)
+                .ok()?
+                .get("realized_profit")?
+                .as_number()?
+                .as_u64()
+        })
+        .sum();
+    let auction_profit: u64 = arbs
+        .iter()
+        .filter_map(|arb_str| arb_str.as_str())
+        .filter(|arb_str| arb_str.contains("auction"))
+        .filter_map(|arb_str| serde_json::from_str::<Value>(arb_str).ok())
+        .filter_map(|arb| arb.get("realized_profit")?.as_number()?.as_u64())
+        .sum();
+
+    println!("ARB BOT PROFIT: {profit}");
+    println!("AUCTION BOT PROFIT: {auction_profit}");
+
+    util::assert_err(
+        "500000 + PROFIT_MARGIN > profit > 500000 - PROFIT_MARGIN",
+        500000 + ERROR_MARGIN_PROFIT > profit && profit > 500000 - ERROR_MARGIN_PROFIT,
+        true,
+    )?;
+    util::assert_err(
+        "500000 + PROFIT_MARGIN > auction_profit > 500000 - PROFIT_MARGIN",
+        500000 + ERROR_MARGIN_PROFIT > auction_profit
+            && auction_profit > 500000 - ERROR_MARGIN_PROFIT,
+        true,
+    )?;
+
+    Ok(())
+}
+
+pub fn test_unprofitable_arb(arbfile: Value) -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    let arbs = arbfile.as_array().expect("no arbs in arbfile");
+
+    util::assert_err("!arbs.is_empty()", arbs.is_empty(), false)?;
```

Comment:
ditto

Reply:
Don't think this is necessary to change.